pkg/cmd/cli/config/store.go

```diff
@@ -116,13 +116,15 @@ func (c *ConfigStore) SaveToFile(credentialsName string, namespace string, clien
 
 	credentials := clientcmdapi.NewAuthInfo()
 	credentials.Token = clientCfg.BearerToken
-	credentials.ClientCertificate = clientCfg.TLSClientConfig.CertFile
-	if len(credentials.ClientCertificate) == 0 {
-		credentials.ClientCertificateData = clientCfg.TLSClientConfig.CertData
-	}
-	credentials.ClientKey = clientCfg.TLSClientConfig.KeyFile
-	if len(credentials.ClientKey) == 0 {
-		credentials.ClientKeyData = clientCfg.TLSClientConfig.KeyData
+	if len(credentials.Token) == 0 {
```

Comment:
You can have a config file with tokens and client certs, but each `user` entry in the config file can only have one authentication method, either client certificates, token, credentials, etc. E.g.:

```
users:
- name: admin
  user:
    client-certificate: /path/to/cert.crt
    client-key-data: /path/to/key.key
- name: regularuser
  user:
    token: OTNiODI3MmQtZTM5YS00MzU4LThmNTAtYmExYjQwNjlkMGE2
```

AFAIK this is not valid:

```
users:
- name: admin
  user:
    client-certificate: /path/to/cert.crt
    client-key-data: /path/to/key.key
    token: OTNiODI3MmQtZTM5YS00MzU4LThmNTAtYmExYjQwNjlkMGE2
```